pkgs/tools/security/sequoia/default.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv, fetchFromGitLab, lib
+, git, nettle, llvmPackages_8, cargo, rustc
+, rustPlatform, pkgconfig, glib
+, openssl, sqlite, capnproto
+, pythonSupport ? true, python37Packages ? null
+}:
+
+assert pythonSupport -> python37Packages != null;
+
+rustPlatform.buildRustPackage rec {
+  version = "0.9.0";
+  pname = "sequoia";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.com";
+    owner = "sequoia-pgp";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "13dzwdzz33dy2lgnznsv8wqnw2501f2ggrkfwpqy5x6d1kgms8rj";
+  };
+  cargoSha256 = "1zcnkpzcar3a2fk2rn3i3nb70b59ds9fpfa44f15r3aaxajsdhdi";
+
+  nativeBuildInputs = [
+    pkgconfig
+    cargo
+    rustc
+    git
+    llvmPackages_8.libclang
+    llvmPackages_8.clang
+  ]
+    ++ lib.optional pythonSupport python37Packages.cffi
+    ++ lib.optional pythonSupport python37Packages.pytest
```

Comment:
this is typically a test dependency and should go to `checkInputs`

Reply:
That should be correct only if I use `make check` as the `checkPhase` instead of the `rustPlatform`'s `buildRustPackage`, thanks. Please see my changes at the new PR: https://github.com/NixOS/nixpkgs/pull/65724